### ISSUE TITLE
Remove the extra dot from the language_message text entry of the dashboard game options JSON

### ISSUE THREAD

**Issue by AerunDev** [PokemonWorkshop/PokemonStudio#326]
There is an extra dot "." in the `language_message` text entry of the `dashboard_game_options` JSON. We should remove it.

**Comment by mathys-mt** [comment]
Hi, this issue seems fixed in the source code : 
  "language_message": "This option is managed automatically.",
Does it still need to be fixed ?

**Comment by AerunDev** [comment]
Yes, it's all done, thanks for the update. I'm closing the issue.
